src/components/Q29Percentage/index.tsx

```diff
@@ -0,0 +1,10 @@
+interface IQ29PercentageProps {
+    value: number;
+    value1: number;
+}
+
+const Q29Percentage: React.FC<IQ29PercentageProps> = ({ value, value1 }) => (
+    <p>{(value / value1) * 100}%</p>
+);
```

Comment:
Se `value1` for zero, vai dar erro de `infinite`. Você entregou essa questão sem testar